src/features/war/components/Leaderboard.tsx

```diff
@@ -0,0 +1,217 @@
+        transition: "all .3s ease-in-out",
+        left: "50%",
+      }}
+    >
+      <Button
+        className={`w-12 h-6  -mt-2 absolute left-1/2 -translate-x-1/2 ${
+          isOpen ? "-top-5" : "-top-7"
+        } z-10`}
+        onClick={() => setIsOpen(!isOpen)}
+      >
+        <div className="relative w-7">
+          <img
+            src={sword}
+            alt=""
+            className="absolute"
+            style={{
+              width: "18px",
+              left: "50%",
+              top: "50%",
+              transform: "translate(-50%, -50%)",
+            }}
+          />
+          <img
+            src={sword}
+            alt=""
+            className="absolute left-1/2 -translate-x-1/2 top-1"
+            style={{
+              width: "18px",
+              left: "50%",
+              top: "50%",
+              transform: "translate(-50%, -50%) scaleX(-1)",
+            }}
+          />
+        </div>
+      </Button>
+      <Panel>
+        <div className="flex justify-between w-full px-8 mb-16">
+          <div className={"flex h-fit"}>
+            <img
+              src={femaleHuman}
+              style={{
+                width: `${PIXEL_SCALE * 13}px`,
+                height: `${PIXEL_SCALE * 20}px`,
+              }}
+              alt=""
+            />
+            <img
+              src={maleHuman}
+              alt=""
+              style={{
+                width: `${PIXEL_SCALE * 13}px`,
+                height: `${PIXEL_SCALE * 20}px`,
+              }}
+            />
+          </div>
+          <div className="absolute w-14 left-1/2 -translate-x-1/2 top-11">
+            <img
+              src={sword}
+              alt=""
+              className="absolute"
+              style={{
+                width: "36px",
+                left: "50%",
+                top: "50%",
+                transform: "translate(-50%, -50%)",
+              }}
+            />
+            <img
+              src={sword}
+              alt=""
+              className="absolute left-1/2 -translate-x-1/2 top-1"
+              style={{
+                width: "36px",
+                left: "50%",
+                top: "50%",
+                transform: "translate(-50%, -50%) scaleX(-1)",
+              }}
+            />
+          </div>
+          <div
+            className="flex items-end"
+            style={{
+              transform: "scaleX(-1)",
+            }}
+          >
+            <img
+              src={femaleGoblin}
+              alt=""
+              style={{
+                width: `${PIXEL_SCALE * 16}px`,
+                height: `${PIXEL_SCALE * 18}px`,
+              }}
+            />
+            <img
+              src={maleGoblin}
+              alt=""
+              style={{
+                width: `${PIXEL_SCALE * 16}px`,
+                height: `${PIXEL_SCALE * 18}px`,
+              }}
+            />
+          </div>
+        </div>
```

Comment:
Looking so good!

**suggestion:** What do you think about removing the `width` or `height` on the humans and goblins? I think they are looking a little bit squashed when both `width` and `height` is specified. If only `width` or `height` is specified, then the other will scale with the right ratio.

In the example below I removed the `width`:

Before:
<img width="373" alt="" src="https://user-images.githubusercontent.com/41215134/188769786-77c56bf9-ed42-492f-ae5d-565dadc7732a.png">

After:
<img width="380" alt="" src="https://user-images.githubusercontent.com/41215134/188769798-18b4e5b3-22fe-4890-879c-8e07e060bd9d.png">


```suggestion
        <div className="flex justify-between w-full px-8 mb-16">
          <div className={"flex h-fit"}>
            <img
              src={femaleHuman}
              style={{
                height: `${PIXEL_SCALE * 20}px`,
              }}
              alt=""
            />
            <img
              src={maleHuman}
              alt=""
              style={{
                height: `${PIXEL_SCALE * 20}px`,
              }}
            />
          </div>
          <div className="absolute w-14 left-1/2 -translate-x-1/2 top-11">
            <img
              src={sword}
              alt=""
              className="absolute"
              style={{
                width: "36px",
                left: "50%",
                top: "50%",
                transform: "translate(-50%, -50%)",
              }}
            />
            <img
              src={sword}
              alt=""
              className="absolute left-1/2 -translate-x-1/2 top-1"
              style={{
                width: "36px",
                left: "50%",
                top: "50%",
                transform: "translate(-50%, -50%) scaleX(-1)",
              }}
            />
          </div>
          <div
            className="flex items-end"
            style={{
              transform: "scaleX(-1)",
            }}
          >
            <img
              src={femaleGoblin}
              alt=""
              style={{
                height: `${PIXEL_SCALE * 18}px`,
              }}
            />
            <img
              src={maleGoblin}
              alt=""
              style={{
                height: `${PIXEL_SCALE * 18}px`,
              }}
            />
          </div>
        </div>
```